toolchain/check/interface.cpp

```diff
@@ -99,20 +156,34 @@ auto GetTypeForSpecificAssociatedEntity(Context& context, SemIRLoc loc,
     -> SemIR::TypeId {
   auto decl =
       context.insts().Get(context.constant_values().GetConstantInstId(decl_id));
-
-  auto specific_id = interface_specific_id;
   if (auto assoc_const = decl.TryAs<SemIR::AssociatedConstantDecl>()) {
-    specific_id = GetSelfSpecificForInterfaceMemberWithSelfType(
-        context, loc, interface_specific_id,
-        context.associated_constants()
-            .Get(assoc_const->assoc_const_id)
-            .generic_id,
-        self_type_id, self_witness_id);
+    // Form a specific for the associated constant, and grab the type from
+    // there.
+    auto generic_id = context.associated_constants()
+                          .Get(assoc_const->assoc_const_id)
+                          .generic_id;
+    auto args =
```

Comment:
Minor, but on lines 134 and 114 you call this `arg_ids`. In the `MakeSpecific` overload you call this `args`. Might consider making names consistent.

Reply:
Done.